linerate/model.py

```diff
@@ -32,273 +32,7 @@
-        min_ampacity:
-            :math:`I_\text{min}~\left[\text{A}\right]`. Lower bound for the numerical scheme for
-            computing the ampacity
-        max_ampacity:
-            :math:`I_\text{min}~\left[\text{A}\right]`. Upper bound for the numerical scheme for
-            computing the ampacity
-        tolerance:
-            :math:`\Delta I~\left[\text{A}\right]`. The numerical accuracy of the ampacity. The
-            bisection iterations will stop once the numerical ampacity uncertainty is below
-            :math:`\Delta I`. The bisection method will run for
-            :math:`\left\lceil\frac{I_\text{min} - I_\text{min}}{\Delta I}\right\rceil` iterations.
-
-        Returns
-        -------
-        Union[float, float64, ndarray[Any, dtype[float64]]]
-            :math:`I~\left[\text{A}\right]`. The thermal rating.
-        """
-        I = solver.compute_conductor_ampacity(  # noqa
-            self.compute_heat_balance,
-            max_conductor_temperature=max_conductor_temperature,
-            min_ampacity=min_ampacity,
-            max_ampacity=max_ampacity,
-            tolerance=tolerance,
-        )
-        n = self.span.num_conductors
-        return I * n
-
-    def compute_conductor_temperature(
-        self,
-        current: Ampere,
-        min_temperature: Celsius = -30,
-        max_temperature: Celsius = 150,
-        tolerance: float = 0.5,
-    ) -> Celsius:
-        r"""Use the bisection method to compute the steady state conductor temperature.
-
-        Parameters
-        ----------
-        current:
-            :math:`I_\text{max}~\left[\text{A}\right]`. The current flowing through the conductor.
-        min_temperature:
-            :math:`T_\text{min}~\left[^\circ\text{C}\right]`. Lower bound for the numerical scheme
-            for computing the temperature
-        max_temperature:
-            :math:`T_\text{max}~\left[^\circ\text{C}\right]`. Upper bound for the numerical scheme
-            for computing the temperature
-        tolerance:
-            :math:`\Delta T~\left[^\circ\text{C}\right]`. The numerical accuracy of the
-            temperature. The bisection iterations will stop once the numerical temperature
-            uncertainty is below :math:`\Delta T`. The bisection method will run for
-            :math:`\left\lceil\frac{T_\text{min} - T_\text{min}}{\Delta T}\right\rceil` iterations.
-
-        Returns
-        -------
-        Union[float, float64, ndarray[Any, dtype[float64]]]
-            :math:`I~\left[\text{A}\right]`. The thermal rating.
-        """
-        T = solver.compute_conductor_temperature(
-            self.compute_heat_balance,
-            current=current,
-            min_temperature=min_temperature,
-            max_temperature=max_temperature,
-            tolerance=tolerance,
-        )
-        n = self.span.num_conductors
-        return T / n
+__all__ = ["ThermalModel", "Cigre601", "IEEE738", "Cigre207"]
 
 
 class Cigre601(ThermalModel):
```

Comment:
Should this class and class IEEE738 be refactored to /models ? Will that break backwards compatibility in imports?

Reply:
I don't think it breaks backwards comptability. Will try to move them out!